### PR TITLE
V1.4.0

### PR DESCRIPTION
**Pull request by cbilodeau-mecademic**

### CODE REVIEW

**Comment by gbmecademic** [comment]
Readme doesn't mention the compatibility to R4, I updated it in the master branch but it is not in this one.

Also does this support the Scara now? If yes it should be included in the readme.

**Review by alainbrassardMeca — APPROVED** [review]

**Comment by alainbrassardMeca** [comment]
> Also does this support the Scara now? If yes it should be included in the readme.

@gbmecademic 
The readme does indicate Mcs500 (scara) is supported:
"Update API to support Mcs500"

Are you thinking about something else to add in the readme?

**Comment by acoulombe** [comment]
> > Also does this support the Scara now? If yes it should be included in the readme.
> 
> @gbmecademic The readme does indicate Mcs500 (scara) is supported: "Update API to support Mcs500"
> 
> Are you thinking about something else to add in the readme?

That's indicated in the Release Notes. The README does not mention Mcs500 support under `Supported Robots`

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+![Mecademic](https://github.com/Mecademic/mecademicpy/blob/main/docs/logo/mecademic_logo.png?raw=true  "Mecademic")
 # Mecademic Python API
 
-A python module designed for robot products from Mecademic. The module offers tools that give access to the features of the Mecademic Robots such as MoveLin and MoveJoints available through the TCP/IP text interface. The module can be started from a terminal or a python application and controls the Mecademic products. 
+A python module designed for robot products from Mecademic. The module offers tools that give access to the features of the Mecademic Robots such as MoveLin and MoveJoints available through the TCP/IP text interface. The module can be started from a terminal or a python application and controls the Mecademic products.
 
 #### Supported Robots
 
- * Meca500 R2, R3, R4
+ * Meca500, Mcs500
```
Comment (acoulombe) [inline README.md:8]: nit: use second bullet point for Mcs500 rather than comma separated

**Review by alainbrassardMeca — APPROVED** [review]

**Review by jletendre-meca — APPROVED** [review]

**Review by lxfalardeau — APPROVED** [review]

**Pull request merged**